app/src/test/java/com/douglasbruce/tasky/core/domain/validation/NameValidatorTests.kt

```diff
@@ -12,6 +12,22 @@ internal class NameValidatorTests {
         nameValidator = NameValidator()
     }
 
+    @Test
+    fun `Invalid name, empty returns false with empty error`() {
+        val name = ""
+        val result = nameValidator(name)
```

Comment:
Here, I'd personally prefer a normal function so it's called `nameValidator.validate()` instead of overriding the invoke operator. Otherwise, it reads like a function with a class name.
But, if you prefer your approach, feel free to keep it of course

Reply:
I have updated the code to add a validate method